.github/workflows/build-mudlet.yml

```
@@ -324,6 +337,38 @@ jobs:
         APPLE_PASSWORD: ${{secrets.APPLE_PASSWORD}}
         APPLE_TEAM_ID: ${{secrets.APPLE_TEAM_ID}}
 
+    - name: Upload debug symbols and source to sentry.io
+      if: env.WITH_SENTRY == 'yes'
+      env:
+        SENTRY_AUTH_TOKEN: ${{secrets.SENTRY_AUTH_TOKEN}}
+      run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2046:warning:23:75: Quote this to prevent word splitting

(shellcheck)